rdt/transformers/base.py

```diff
@@ -318,6 +320,19 @@ def _fit(self, columns_data):
         """
         raise NotImplementedError()
 
+    def _set_seed(self, data):
+        hash_value = self.get_input_column()
+        for value in data.head(5):
+            hash_value += str(value)
+
+        hash_value = int(hashlib.sha256(hash_value.encode('utf-8')).hexdigest(), 16)
+        self.random_seed = hash_value % ((2 ** 32) - 1)  # maximum value for a seed
```

Comment:
Since we store the random seed on the object, maybe we should just use a uuid and then we could even get the seed before fitting